src/main/java/reposense/git/GitRemote.java

```diff
@@ -0,0 +1,43 @@
+package reposense.git;
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import reposense.system.CommandRunner;
+
+/**
+ * Contains git remote related functionality.
+ * Git remote is responsible for getting remote repository information.
+ */
+public class GitRemote {
```

Comment:
Not directly related to this PR but this seems like an extremely strange line for codecov to flag as not covered.

Reply:
Oh I assumed that adding the tests would have gotten rid of this. I'm actually not sure why this is flagged.